pages/reference/OS/advanced.md

```diff
@@ -5,94 +5,91 @@ excerpts: Configuration options to expose more device functionality to {{ $names
 
 # Advanced boot settings
 
-__Warning:__ This page contains details of the more *involved* configuration
-options that expose more functionality but potentially enable you to break
-things should you configure them incorrectly - tread carefully!
+__Warning:__ This page contains details of advanced configuration options that expose more functionality, but any mistakes may potentially leave a device inaccessible.
 
 ## Raspberry Pi
 
-### config.txt
+The Raspberry Pi exposes device [configuration options][config-txt] via a text file on the [boot partition][boot-partition] named [`config.txt`][config-txt]. You can change boot options simply by editing this file, either by manually editing before the device's first boot or editing the device [configuration variables][config-vars]. By default, the following values are set for Raspberry Pi devices:
```

Comment:
"simply by editing this file" -> "in this file" ?
"manually editing" -> "manually editing it" ?